src/data/constants.js

```diff
@@ -36,3 +36,13 @@ export const INVALID_NAME_REGEX = /[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{
 // Query string parameters that can be passed to LMS to manage
 // things like auto-enrollment upon login and registration.
 export const AUTH_PARAMS = ['course_id', 'enrollment_action', 'course_mode', 'email_opt_in', 'purchase_workflow', 'next', 'save_for_later', 'register_for_free'];
+
+// Education difficulty level mapping
+export const EDUCATION_LEVEL_MAPPING = {
```

Comment:
could you please guide me on what is p, m, b, a, hs, jhs in it? is it the values that we get from the field? also, I am thinking is there any better way to map characters to the same level to avoid code duplication?

Reply:
Yes. we get these values from education level field and we required this mapping for algolia query. We can reduce this duplication but then we need to write some extra code which will also increase  the complexity.